thanos/charts/templates/ruler/ruler-svc.yaml

```diff
@@ -26,5 +26,5 @@ spec:
     protocol: TCP
     targetPort: http
   selector:
-    app.kubernetes.io/managed-by: {{ include "release.name" . }}
-    app.kubernetes.io/name: ruler
+    app.kubernetes.io/managed-by: prometheus-operator
```

Comment:
Do we actually need to select this? The bottom line should be sufficient, no?

Reply:
won't that also select unwanted pods managed by prometheus-operator ?